pkg/types/types.go

```diff
@@ -197,29 +197,57 @@ type Metric struct {
 }
 
 // MergeMetrics merges metrics by name.
```

Comment:
I think a word or two about what `consistencyCheck` arg means would help a lot to understand its meaning w/o having to read the full implementation.

Reply:
Absolutely.